public/locations-list-map.js

```diff
@@ -460,32 +491,6 @@ function resetMap() {
 
 
 
-/**
- * Takes the form input and attempts to pull geocoded information from it. Once done, centers map on closest markers nearby.
- */
-function loadSearchInMap() {
```

Comment:
Removed this function because I needed the scope from the caller so I could pull the `$search.val()` without duplicating too much code.